docs/develop/documentation.md

```
@@ -21,29 +21,29 @@ Add the 🆕 or 🚧 emoji if applicable
 * [MoonBase docs](https://github.com/MoonModules/MoonLight/tree/main/docs/moonbase)
-* [MoonBase Webpages](https://moonmodules.org/MoonLight/moonbase/overview/)
+* [MoonBase Webpages](https://moonmodules.org/MoonLight/moonbase/overview)
 
 * [MoonLight docs](https://github.com/MoonModules/MoonLight/tree/main/docs/moonlight)
-* [MoonLight Webpages](https://moonmodules.org/MoonLight/moonlight/overview/)
+* [MoonLight Webpages](https://moonmodules.org/MoonLight/moonlight/overview)
 
 ## Images
 
 * Image
 	* For drivers an image of what is driven, for Effects, Modifiers and layouts image showing the result on the monitor. 
```

markdownlint-cli2 (0.18.1)

Unordered list indentation
Expected: 2; Actual: 1

(MD007, ul-indent)